templates/frontend/pages/editorialMasthead.tpl

```diff
@@ -0,0 +1,77 @@
+{**
+ * templates/frontend/pages/editorialMasthead.tpl
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @brief Display context's editorial masthead page.
+ *
+ *}
+{include file="frontend/components/header.tpl" pageTitle="common.editorialMasthead"}
+
+<main class="page page_masthead">
+	<div class="container-fluid container-page container-narrow">
+
+        {include file="frontend/components/headings.tpl" currentTitleKey="common.editorialMasthead"}
+
+        {foreach from=$mastheadRoles item="mastheadRole"}
+			{if array_key_exists($mastheadRole->id, $mastheadUsers)}
+				<h2>{$mastheadRole->getLocalizedData('name')|escape}</h2>
+				<ul class="user_listing" role="list">
+					{foreach from=$mastheadUsers[$mastheadRole->id] item="mastheadUser"}
+						<li>
+							{strip}
+								<span class="date_start">{translate key="common.fromUntil" from=$mastheadUser['dateStart'] until=""}</span>
+								<span class="name">
+								{$mastheadUser['user']->getFullName()|escape}
+									{if $mastheadUser['user']->getData('orcid') && $mastheadUser['user']->hasVerifiedOrcid()}
+										<a class="orcid-image-url" href="{$mastheadUser['user']->getData('orcid')|escape}" target="_blank" aria-label="{translate key="common.editorialHistory.page.orcidLink" name=$mastheadUser['user']->getFullName()|escape}">
+											{$orcidIcon}
+										</a>
+									{/if}
+								</span>
+								{if !empty($mastheadUser['user']->getLocalizedData('affiliation'))}
+									<span class="affiliation">{$mastheadUser['user']->getLocalizedData('affiliation')|escape}</span>
+								{/if}
+							{/strip}
+						</li>
+					{/foreach}
+				</ul>
+			{/if}
+		{/foreach}
+		<hr>
+		<p>
+			{capture assign=editorialHistoryUrl}{url page="about" op="editorialHistory" router=\PKP\core\PKPApplication::ROUTE_PAGE}{/capture}
+			{translate key="about.editorialMasthead.linkToEditorialHistory" url=$editorialHistoryUrl}
+		</p>
```

Comment:
We have recently decided to move this paragraph in front of the <hr> (separation line) -- I have changed it for core, but have still not made the PR (coming soon), so you can change it here (and in other themes as well.